lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -116,7 +116,7 @@ def extract_forwardable_args(args)
         end
 
         def only_forwards_all?(send_classifications)
-          send_classifications.each_value.all? { |c, _, _| c == :all }
+          send_classifications.each.all? { |_, (c, _, _)| c == :all }
```

Comment:
It seems possible to simplify the block arguments.

```suggestion
          send_classifications.to_h.each_values.all? { |c, _, _| c == :all }
```

Reply:
I ended up with a slightly different simplification - I realised I don't need the nested array structure, so can remove the array destructuring in all of the `send_classifications` iteration block args